src/console/Command.cpp

```diff
@@ -0,0 +1,103 @@
+#include "console/Command.hpp"
+#include "storm/Error.hpp"
+
+// Extern
+TSHashTable<CONSOLECOMMAND, HASHKEY_STRI> g_consoleCommandHash;
+char g_commandHistory[HISTORY_DEPTH][CMD_BUFFER_SIZE];
+uint32_t g_commandHistoryIndex;
```

Comment:
I know these have `g_` prefixes suggesting `extern`, but are we currently aware of any uses outside of `Command.cpp`? If not, we may consider dropping the `extern` from `Command.hpp`, and keeping them local to the translation unit.

Reply:
I'm not aware of any use outside of the Command translation unit either, but I think it was designed that way since there is otherwise no getter function for `g_consoleCommandHash`. I'll let you make the call on this one.

Reply:
We can leave these as externs for now.